integration-tests/main_test.go

```diff
@@ -22,8 +22,8 @@ var options struct {
 }
 
 func TestMain(m *testing.M) {
-	flag.StringVar(&options.clientSecret, "client-secret", "", "the client secret for the portal app")
-	flag.StringVar(&options.clientID, "client-id", "", "the client id for the portal app")
+	flag.StringVar(&options.clientSecret, "client-secret", "HE1-0PF98Q6P5o-zJ_DWUWrt42fUJ7I8KKGMEvM7_6o", "the client secret for the portal app")
+	flag.StringVar(&options.clientID, "client-id", "p1rp9216dgkor", "the client id for the portal app")
```

Comment:
Don't share secrets, please. This values are suppose to be taken from environment variables.

Reply:
Fixed